### PR TITLE
chore: minor correction of repo readme file

### PR DESCRIPTION
**Pull request by nille**
Correction of docker tag directive to be inline with comment

### CODE REVIEW

**Review by niklas-palm — CHANGES_REQUESTED** [review]
Should consistently be studio-cli

File: README.md [hunk 1]
```diff
@@ -24,8 +24,8 @@ pip install .
 For convenience, and to avoid installing the CLI/Python dependencies on your machine you can use Docker to run the CLI.
 
 ```bash
-docker build . -t studio-cli # Build the Docker image and tag it as "hackathon-cli"
-docker run -it --rm studio-cli # Run the CLI in a container (and remove the container when exiting)
+docker build . -t hackathon-cli # Build the Docker image and tag it as "hackathon-cli"
```
Comment (nille) [inline README.md]:
```suggestion
docker build . -t studio-cli # Build the Docker image and tag it as "studio-cli"
```

File: README.md [hunk 2]
```diff
@@ -24,8 +24,8 @@ pip install .
...
+docker run -it --rm hackathon-cli # Run the CLI in a container (and remove the container when exiting)
```
Comment (nille) [inline README.md]:
```suggestion
docker run -it --rm studio-cli # Run the CLI in a container (and remove the container when exiting)
```

File: README.md (hunk 1 above)
Comment (nille) [inline README.md]:
```suggestion
docker build . -t studio-cli # Build the Docker image and tag it as "studio-cli"
```

File: README.md (hunk 2 above)
Comment (nille) [inline README.md]:
```suggestion
docker run -it --rm studio-cli # Run the CLI in a container (and remove the container when exiting)
```

Comment (nille) [inline README.md]:
```suggestion
docker run -it --rm studio-cli # Run the CLI in a container (and remove the container when exiting)
```

**Review by niklas-palm — APPROVED** [review]

**Pull request merged**